core/frontend/src/IModelConnection.ts

```diff
@@ -444,9 +444,9 @@ export abstract class IModelConnection extends IModel {
   }
 
   /** Request a named texture image from the backend.
-   * @param textureLoadProps The texture load properties which must contain a name property (a valid 64-bit integer identifier)
+   * @param textureLoadProps The texture load properties which must contain a name property (a valid 64-bit integer identifier) and the maximum texture size supported by the client.
    * @see [[Id64]]
-   * @alpha
+   * @public

```

Comment:
Should we make this public? Previously alpha...

Reply:
Sure why not?